Packs/Ipstack/Integrations/Ipstack/Ipstack.py

```diff
@@ -5,7 +5,7 @@
 import requests
 
 BASE_URL = 'http://api.ipstack.com'
-API_KEY = demisto.params().get('apikey')
+API_KEY = demisto.params().get('credentials', {}).get('password') or demisto.params().get('apikey')
```

Comment:
`password` overrides `apikey` now?

Reply:
Yes, because the apikey has become hidden, and if it is the first, they will not be able to change it